Resources/views/Calendar/view.html.twig

```diff
@@ -119,7 +119,15 @@
                             <hr/>
                         </div>
                         {% endif %}
-                        <div><label class="bold">{{'global.stop_point_code'|trans({}, 'messages')}}</label>: {{ calendar.schedules.stop_point.codes|externalCode }}</div>
+                        <div>
+                            <dl>
+                                <dt>{{'global.stop_point_code'|trans({}, 'messages')}}</dt>
+                                {% for code in calendar.schedules.stop_point.codes %}
+                                {% set value = code.type == 'external_code' ? code.value|slice(3) : code.value %}
```

Comment:
Maybe there is an obstacle for put the view model pattern thaht I ignore,  but this case can't be the oportunity to insert view model for this specific point, a refactoring could be done step by step, in a second time ?
I imagine In a first time thtat View Model just get all parameters passed to view in the controller. 
What do you think about that ?